Fix build with systemd v233.

The new systemd provides united libsystemd instead of old libsystemd-login.
The fix is based on 2291767a014f5a04a92ca6f0eb472794f212ca67 upstream commit.

diff -purN polkit/configure.ac polkit-izh/configure.ac
--- polkit/configure.ac	2020-05-01 14:23:48.027758476 +0000
+++ polkit-izh/configure.ac	2020-05-01 14:43:41.438236473 +0000
@@ -158,14 +158,18 @@ AC_ARG_ENABLE([systemd],
               [enable_systemd=auto])
 if test "$enable_systemd" != "no"; then
   PKG_CHECK_MODULES(SYSTEMD,
-                    [libsystemd-login],
+                    [libsystemd],
                     have_systemd=yes,
-                    have_systemd=no)
+                    dnl if libsystemd is not available, fall back to the older libsystemd-login
+                    [PKG_CHECK_MODULES(SYSTEMD,
+                      [libsystemd-login],
+                        have_systemd=yes,
+                        libsystemd=no)])
   if test "$have_systemd" = "yes"; then
     SESSION_TRACKING=systemd
   else
     if test "$enable_systemd" = "yes"; then
-      AC_MSG_ERROR([systemd support requested but libsystemd-login1 library not found])
+      AC_MSG_ERROR([systemd support requested but libsystemd or libsystemd-login1 library not found])
     fi
   fi
 fi
